src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -65,6 +107,23 @@ public void setUp() {
     @DisplayName("지하철 노선을 조회한다.")
     @Test
     void 지하철_노선_조회_테스트() {
+        // given
+        Station station1 = stations.save(new Station("신사역"));
+        Station station2 = stations.save(new Station("광교중앙"));
+
+        Map<String, String> params = setParams(new LineRequest("신분당선", "bg-red-600", station1.getId(), station2.getId(), 10L));
+
+        ExtractableResponse<Response> response = createLine(params);
+        String lineId = parseIdByLocation(response.header("Location"));
+
+        // when
+        LineResponse lineResponse = retrieveLineById(lineId);
```

Comment:
```suggestion
        String lineLocation = response.header("Location");

        // when
        LineResponse lineResponse = retrieveLine(lineLocation);
```
`Location`에서 id만을 파싱하고 다시 URL을 만들지말고
`Location`을 그대로 URL로 사용하는건 어떨까요?